app/test_split_integer.py

```diff
@@ -1,21 +1,29 @@
+import pytest
+
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 17
+    number_of_parts = 4
+    assert sum(split_integer(17, 4)) == value
```

Comment:
Code Style: You have created unnecessary variables `value` and `number_of_parts`. You can directly use the values in the function call and the assert statement. It would look like this: `assert sum(split_integer(17, 4)) == 17`. This would make your code cleaner and easier to read.